pygeoapi/plugin.py

```diff
@@ -52,6 +52,7 @@
         'MongoDB': 'pygeoapi.provider.mongo.MongoProvider',
         'MVT-tippecanoe': 'pygeoapi.provider.mvt_tippecanoe.MVTTippecanoeProvider',  # noqa: E501
         'MVT-elastic': 'pygeoapi.provider.mvt_elastic.MVTElasticProvider',  # noqa: E501
+        'MVT-postgis': 'pygeoapi.provider.mvt_postgis.MVTPostGISProvider',  # noqa: E501
```

Comment:
Can we change this name to a more generic `'MVT-proxy': 'pygeoapi.provider.mvt_proxy.MVTProxyProvider'` and ensure that in the docs the supported targets like `martin` and `pg_tileserv` are listed?

Reply:
Thanks Francesco, I updated the code and the documentation following your suggestion.